backend/get_past_matches.js

```diff
@@ -0,0 +1,16 @@
+const sdk = require('api')('@developers-pandascore/v2#apvn9q1ml4sw7gei');
+import * as dotenv from 'dotenv'
+
+dotenv.config(({path:__dirname+'/.env'}))
+
+sdk.auth(process.env.APIKEY);
+
+sdk['get_matches_past']({
```

Comment:
Why not `sdk.get_matches_past`?

Reply:
True, that would be more proper
Change done 👍🏻